Sources/Component/CollapsibleView/CollapsibleViewModel.swift

```diff
@@ -5,37 +5,47 @@ import UIKit
 /// The state view model for `CollapsibleView`.
 public struct CollapsibleViewModel<HeaderViewModel: ViewModelProtocol>: ViewModelProtocol {
     /// The header view to embed.
-    public let headerViewModel: HeaderViewModel
+    public var headerViewModel: HeaderViewModel
     /// The items to add to the collapsible view.
-    public let items: [UIView]
-    /// The state of the collapsible view
-    public var isCollapsed: Bool
+    public var items: [UIView]
+    /// The state of the collapsible view when it is first presented
+    public let isInitiallyCollapsed: Bool
+    /// Whether to animate the collapsing and expanding of the content items
+    public var isAnimated: Bool
     /// The animation duration for the state change of the collapsible view
-    public let animationDuration: TimeInterval
+    public var animationDuration: TimeInterval
     /// The distribution of the content items in the collapsible view
-    public let contentDistribution: UIStackView.Distribution
+    public var contentDistribution: UIStackView.Distribution
     /// The alignment of the content items in the collapsible view
-    public let contentAlignment: UIStackView.Alignment
+    public var contentAlignment: UIStackView.Alignment
+    /// The closure to call when the collapsible state changes.
     public var didChangeCollapsibleState: ((Bool) -> Void)?
 
     /// The default initializer of `CollapsibleViewModel`.
     ///
     /// - Parameter headerViewModel: The model of the header view.
     /// - Parameter items: The items to add to the collapsible view.
-    /// - Parameter isCollapsed: The state of the collapsible view
+    /// - Parameter isInitiallyCollapsed: The state of the collapsible view when it is first presented.
+    ///             This will not be updated by the `CollapsibleView` on subsequent state changes.
+    /// - Parameter isAnimated: Whether to animate the collapsing and expanding of the content items
     /// - Parameter animationDuration: The animation duration for the state change of the collapsible view
+    /// - Parameter contentDistribution: The distribution of the content items in the collapsible view
+    /// - Parameter contentAlignment: The alignment of the content items in the collapsible view
+    /// - Parameter didChangeCollapsibleState: The closure to call when the collapsible state changes.
     public init(
         headerViewModel: HeaderViewModel = Self.default.headerViewModel,
         items: [UIView] = Self.default.items,
-        isCollapsed: Bool = Self.default.isCollapsed,
+        isInitiallyCollapsed: Bool = Self.default.isInitiallyCollapsed,
```

Comment:
I would prefer to not rename the parameter, since this would break implementations that are using this parameter after update.

Reply:
I re-added an initializer with the old `isCollapsed` label and marked it as deprecated. This way initializing the view model on dependent implementations would still work.

Reading or manually setting the `isCollapsed` property will still cause compiler errors after updating JamitFoundation, but in my opinion, this is fine, as reading or manually changing the `isCollapsed`/`isInitiallyCollapsed` property after the first initialization of the view model has no effect now and should be avoided.